### PR TITLE
resource_manager: add metrics to show resource group configuration

### PR DESCRIPTION
**Pull request by nolouch**
### What problem does this PR solve?

Issue Number: Close https://github.com/tikv/pd/issues/8444

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
image

### Release note

```release-note
None.
```

### CODE REVIEW

**Comment by nolouch** [comment]
PTAL @JmPotato @HuSharp

**Review by JmPotato — APPROVED** [review]

File: pkg/mcs/resourcemanager/server/manager.go [hunk 1]
```diff
@@ -48,6 +48,11 @@ const (
 
 	reservedDefaultGroupName = "default"
 	middlePriority           = 8
+
+	// Labels for the metrics.
+	ruPerSec   = "ru_per_sec"
```
Comment (HuSharp) [inline pkg/mcs/resourcemanager/server/manager.go:53]: why not put them in `metrics.go`?

File: pkg/mcs/resourcemanager/server/metrics.go [hunk 2]
```diff
@@ -116,6 +116,14 @@ var (
...
 			Help:      "Counter of the available RU for all resource groups.",
 		}, []string{resourceGroupNameLabel, newResourceGroupNameLabel})
+
+	resourceGroupConfigGauge = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: serverSubsystem,
+			Name:      "group_config",
+			Help:      "Config of the resource group.",
+		}, []string{newResourceGroupNameLabel, "type"})
```
Comment (HuSharp) [inline pkg/mcs/resourcemanager/server/metrics.go:126]:
```suggestion
		}, []string{newResourceGroupNameLabel, typeLabel})
```

**Review by HuSharp — APPROVED** [review]

**Pull request merged**